diff --git a/node_modules/lenis/dist/lenis.js b/node_modules/lenis/dist/lenis.js
index 661ec30..ef418ea 100644
--- a/node_modules/lenis/dist/lenis.js
+++ b/node_modules/lenis/dist/lenis.js
@@ -318,6 +318,7 @@
 		* @param event Touch event
 		*/
 		onTouchStart = (event) => {
+			if (event.targetTouches && event.targetTouches.length === 0) return;
 			const { clientX, clientY } = event.targetTouches ? event.targetTouches[0] : event;
 			this.touchStart.x = clientX;
 			this.touchStart.y = clientY;
@@ -333,6 +334,7 @@
 		};
 		/** Event handler for 'touchmove' event */
 		onTouchMove = (event) => {
+			if (event.targetTouches && event.targetTouches.length === 0) return;
 			const { clientX, clientY } = event.targetTouches ? event.targetTouches[0] : event;
 			const deltaX = -(clientX - this.touchStart.x) * this.options.touchMultiplier;
 			const deltaY = -(clientY - this.touchStart.y) * this.options.touchMultiplier;
diff --git a/node_modules/lenis/dist/lenis.mjs b/node_modules/lenis/dist/lenis.mjs
index c36651d..0ea6434 100644
--- a/node_modules/lenis/dist/lenis.mjs
+++ b/node_modules/lenis/dist/lenis.mjs
@@ -309,6 +309,7 @@ var VirtualScroll = class {
 	* @param event Touch event
 	*/
 	onTouchStart = (event) => {
+		if (event.targetTouches && event.targetTouches.length === 0) return;
 		const { clientX, clientY } = event.targetTouches ? event.targetTouches[0] : event;
 		this.touchStart.x = clientX;
 		this.touchStart.y = clientY;
@@ -324,6 +325,7 @@ var VirtualScroll = class {
 	};
 	/** Event handler for 'touchmove' event */
 	onTouchMove = (event) => {
+		if (event.targetTouches && event.targetTouches.length === 0) return;
 		const { clientX, clientY } = event.targetTouches ? event.targetTouches[0] : event;
 		const deltaX = -(clientX - this.touchStart.x) * this.options.touchMultiplier;
 		const deltaY = -(clientY - this.touchStart.y) * this.options.touchMultiplier;
